crates/ra_syntax/src/grammar.ron

```diff
@@ -23,8 +25,9 @@ Grammar(
         ["^", "CARET"],
         ["%", "PERCENT"],
     ],
+    // TODO: Confirm surmision: the tokens which cannot be recorded in a single UTF-8 byte
     multi_byte_tokens: [
-        [".", "DOT"],
+        [".", "DOT"], // Note: DOT is here because <TODO: REASON>
```

Comment:
Hm, it might be just an accident actually

Reply:
OK! Should I change it?

Reply:
Yep, let's try to change it it usual token and see if anything breaks!